app/controllers/foreman_tasks/tasks_controller.rb

```diff
@@ -103,6 +92,19 @@ def resource_class
 
     private
 
+    def process_index(scope)
```

Comment:
slightly better name would be `process_list` or `response_with_tasks`, since sub_tasks is not index

Reply:
Good point, fixed.